frontend/sac-mobile/components/dropdown.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState} from 'react';
+import {ScrollView, Text, StyleSheet, DimensionValue} from 'react-native';
+import { Dropdown } from 'react-native-element-dropdown';
+
+// Library Component
+// https://www.npmjs.com/package/react-native-element-dropdown?activeTab=code
+
+type Item = {
+    label: string, 
+    value: string,
+}
+
+type ListOfItem = {
+    title: string,
+    item: Array<Item>,   // list of dropdown items
+    placeholder: string, // placeholder
+    onChangeText: (...event: any[]) => void;
+    value: Item;
+    onSubmitEditing: () => void;
+    search?: boolean; // true for enable search
+    height?: DimensionValue;
+    error?: boolean;
+}
+
+export const DropdownComponent = (props: ListOfItem) => {
+  const [isFocus, setIsFocus] = useState(false);
+  const borderColor = props.error ? 'red' : 'black';
+  const borderWidth = props.error ? 1 : 0.5;
+
+  const styles = StyleSheet.create({
+    container: {
+      backgroundColor: 'white',
+      height: props.height || 78,
+    },
+    dropdown: {
+      height: '85%',
+      borderColor: borderColor,
+      borderWidth: borderWidth,
+      borderRadius: 12,
+      paddingHorizontal: '5%',
+    },
+    placeholderStyle: {
+      fontSize: 14,
+      color: '#CDCBCB',
```

Comment:
magic numbers / styles are not ideal which is why our goal is to move away from this and use the tailwind.config.js before we have too many styles